strr-web/composables/useChipFlavour.ts

```diff
@@ -46,27 +46,28 @@ export const useChipFlavour = () => {
       case ApplicationStatusE.PAID:
       case HostApplicationStatusE.PAID:
       case ExaminerApplicationStatusE.PAID:
-        return examinerOrHostStatusMap(AlertsFlavourE.APPLIED, 'paid')
+        return examinerOrHostStatusMap(AlertsFlavourE.APPLIED, 'paid', 'paid')
```

Comment:
This is a bit confusing. Can you please add some comments for context, why we use `paid` `paid` twice? And duplicate on all others

Reply:
I modified `examinerOrHostStatusMap` to accept `examinerKey` and `hostKey`. `paid` is used twice as it is the same for `examinerKey` and `hostKey`. 
Right now, only the following condition requires different keys.
https://github.com/bcgov/STRR/blob/35b682bcaceb43cfb689df0c1e6dc2101a544ed4/strr-web/composables/useChipFlavour.ts#L59

Reply:
Can you make the prop optional (at least for now), and use only for the instance where it is different. We can refactor after.